apps/evm/src/hooks/useOperationModal/Modal/BorrowForm/__testUtils__/fakeData.ts

```diff
@@ -16,3 +17,11 @@ export const fakeAsset = {
   userWalletBalanceTokens: new BigNumber(10000000),
   tokenPriceCents: new BigNumber(100),
 };
+
+export const fakeWEthAsset: Asset = {
```

Comment:
Nit - since we write `weth`:

```suggestion
export const fakeWethAsset: Asset = {
```